src/MBC7.md

```diff
@@ -40,6 +40,12 @@ registers. Reads return $FF. Other writes do not appear to do anything
 (Partially unconfirmed). Note that you cannot re-latch the accelerometer
 value without first erasing it; attempts to do so yield no change.
 
+Once the latch for a sample update has been triggered the cpu should not be
+put into HALT for at least 1.2 msec since HALTing turns off the PHI cartridge
+signal (clock from the GB) which the MBC7 accelerometer sensor relies on.
```

Comment:
```suggestion
signal (the System Clock) which the MBC7 accelerometer sensor relies on.
```

Reply:
PHI clock runs at a different speed than the System Clock, so they are not interchangeable. It might be described as "a clock signal from the CPU" 

From elsewhere in the pandocs:
https://gbdev.io/pandocs/Gameboy_Camera.html
> The Game Boy Camera cartridge is one of the few cartridges that use the PHI signal (clock from the GB). That signal is a 1 MiHz clock (1047567 Hz). 

I guess that might mean this part of the Pandocs needs updating:
https://gbdev.io/pandocs/External_Connectors.html?highlight=PHI#cartridge-slot
> `2	PHI	System Clock`

Reply:
Having it inconsistent with how PHI is described at its respective page would've been confusing. Let's update it at the PHI definitions and then propagate the change once that change is discussed/applied

Reply:
Could you please file an Issue regarding this? @bbbbbr

Reply:
Sounds like a good idea. Probably good to confirm that PHI is 2MHz in GBC double speed mode (I see it implied in some places, but don't see it explicitly documented).

Reply:
>  Let's update it at the PHI definitions and then propagate the change once that change is discussed/applied

Looking over pandocs, maybe PHI could remain as System Clock if it's speed is indicated as 1MHz and the 4MHz clock has a distinct name as Master Clock or Oscillator Clock. Right now there is a mix of things in various places which for me, at least, leads to confusion.

Here's a PR that attempts to clarify that #597